src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -203,17 +204,19 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
             [FromQuery]
             Models.DumpType type = Models.DumpType.WithHeap,
             [FromQuery]
-            string egressProvider = null)
+            string egressProvider = null,
+            [FromQuery]
+            Models.PackageMode mode = Models.PackageMode.None)
```

Comment:
nit: We use a variable called `mode` here in reference to a `PackageMode` and `type` for a `DumpType`; however, in the `DumpAsync` method we use `mode` to refer to a `DumpType` and use `packageMode` for a `PackageMode`. Just for consistency and readability, we should probably use the same terms in both places if they mean the same thing.